code/__HELPERS/unsorted.dm

```diff
@@ -2072,7 +2072,7 @@ GLOBAL_DATUM_INIT(dview_mob, /mob/dview, new)
 	return _list
 
 /// Waits at a line of code until X is true
-#define UNTIL(X) while(!(X)) stoplag()
+#define UNTIL(X) while(!(X)) sleep(world.tick_lag)
```

Comment:
Before someone cries "REEE UNDOCUMENTED CHANGE"

This has been on my todo list, and is actually a bugfix. `UNTIL` is meant to just wait until the next tick, not 10 seconds for the lag to stop.